salt/modules/chocolatey.py

```diff
@@ -106,7 +106,7 @@ def chocolatey_version():
         return __context__['chocolatey._version']
 
     cmd = [_find_chocolatey(__context__, __salt__)]
-    cmd.append('-v')
```

Comment:
I was curious about that, it seemed to run faster (once) with -v than with --version, but I wasn't timing it so I could have been hallucinating.